Selenium.Axe/Selenium.Axe/AxeResult.cs

```diff
@@ -31,6 +31,8 @@ public AxeResult(JObject result)
             Inapplicable = inapplicableToken?.ToObject<AxeResultItem[]>();
             Incomplete = incompleteToken?.ToObject<AxeResultItem[]>();
             Timestamp = timestampToken?.ToObject<DateTimeOffset>();
+
+            //AxeRunOptions
```

Comment:
Did you mean to remove line 26 above?

Reply:
Well that is really not what I wanted to do.

Reply:
Fixed